.github/workflows/ci.yml

```diff
@@ -85,6 +86,37 @@ jobs:
             .github/workflows/ci-worker.yml
             .github/workflows/build-worker.yml
             .github/workflows/deploy-worker-*.yml
+      - name: changed files for editor - core
+        id: editor-core
+        uses: step-security/changed-files@3dbe17c78367e7d60f00d78ae6781a35be47b4a1 # v45.0.1
+        with:
+          since_last_remote_commit: true
+          files: |
+            editor/core/**
+            .github/workflows/ci-editor.yml
+            .github/workflows/build-editor.yml
+            .github/workflows/deploy-editor-*.yml
+      - name: changed files for editor - server
+        id: editor-server
+        uses: step-security/changed-files@3dbe17c78367e7d60f00d78ae6781a35be47b4a1 # v45.0.1
+        with:
+          since_last_remote_commit: true
+          files: |
+            editor/server/**
+            .github/workflows/ci-editor.yml
+            .github/workflows/build-editor.yml
+            .github/workflows/deploy-editor-*.yml
+      - name: changed files for editor - web
+        id: editor-web
+        uses: step-security/changed-files@3dbe17c78367e7d60f00d78ae6781a35be47b4a1 # v45.0.1
+        with:
+          since_last_remote_commit: true
+          files: |
+            editor/web/**
+            .github/workflows/ci-editor.yml
+            .github/workflows/build-editor.yml
+            .github/workflows/deploy-editor-*.yml
+
```

Comment:
@pyshx @soneda-yuya I splitted the trigger based on the changes on each section. So the triggers are isolated per component. 
Please let me know if changes on 1 component ( web, server, core) should trigger CI on all instead